pkg/api/v1/resource/helpers.go

```diff
@@ -24,36 +24,25 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
+	corev1helpers "k8s.io/component-helpers/scheduling/corev1"
 )
 
 // PodRequestsAndLimits returns a dictionary of all defined resources summed up for all
-// containers of the pod. Pod overhead is added to the
-// total container resource requests and to the total container limits which have a
-// non-zero quantity.
+// containers of the pod. Pod overhead is added to the total container resource requests
+// and to the total container limits which have a non-zero quantity.  This may not be accurate for
+// scheduled pods as it does not take the InPlacePodVerticalScaling feature gate into consideration.
 func PodRequestsAndLimits(pod *v1.Pod) (reqs, limits v1.ResourceList) {
```

Comment:
These functions aren't used internally any longer, preferring the corev1helpers versions.

Reply:
The code is re-directing to call PodRequestsAndLimitsReuse, then call corev1helpers version.